Makefile

```diff
@@ -254,6 +253,8 @@ bundle: update-mgr-env manifests kustomize operator-sdk rename-csv build-prometh
 		--patch '[{"op": "replace", "path": "/spec/replaces", "value": "$(REPLACES)"}]'
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --package $(BUNDLE_PACKAGE) --version $(OPERATOR_VERSION) $(BUNDLE_METADATA_OPTS) \
 		--extra-service-accounts topolvm-node,vg-manager,topolvm-controller
+	# now we remove the createdAt annotation as otherwise our change detection will say that a the bundle changed during verification
+	sed -i '/createdAt/d' ./bundle/manifests/$(BUNDLE_PACKAGE).clusterserviceversion.yaml
```

Comment:
This does not work on MacOS. Could you adapt it like [here](https://github.com/openshift/lvm-operator/blob/main/Makefile#L71)?